### ISSUE TITLE
Buttons desapear on youtube video

### ISSUE THREAD

**Issue by Arthur-Barreto** [brave/brave-browser#41040]
### Description

![image](https://github.com/user-attachments/assets/eda9fe05-0375-4438-a8f4-4bf5d1402eba)

Buttons of like/dislike, share ... are not visible

### Steps to reproduce

1. install brave on popos 22.04
2.1.69.162 Chromium: 128.0.6613.120 (Official Build) unknown (64-bit) 
3. what is the YouTube video: https://www.youtube.com/watch?v=fiYL5CdvjZU

### Actual result

![image](https://github.com/user-attachments/assets/d06ce05c-60b3-43df-a216-b88b140f7fcf)

I can't see the buttons, as shown in the picture.

Even if I disable the shields, it continues

![image](https://github.com/user-attachments/assets/ecf725ea-2d3a-4a82-ae8c-602806eddb9d)

### Expected result

The buttons should be visible 

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

1.69.162 Chromium: 128.0.6613.120 (Official Build) unknown (64-bit) 

### Channel information

- [X] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [X] with Brave Shields disabled
- [X] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by ayush475** [comment]
This is not a brave issue , I can't seem to reproduce this

**Comment by Arthur-Barreto** [comment]
No, I don't have. I've updated my brave, then the issue was solved.
